test/utils/Utils.t.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.23;
+
+import { Test, console2 } from "forge-std/Test.sol";
+import { Utils } from "script/contracts/Utils.sol";
+
+contract UtilsTest is Test {
+    Utils utils;
+
+    function setUp() public {
+        vm.setEnv("NETWORK", "testnet");
+
+        utils = new Utils();
+    }
+
+    function test_readAndWriteL1AddressesFile() public {
+        Utils.L1AddressesConfig memory config =
+            Utils.L1AddressesConfig({ L1LiskToken: address(0x1), L1VestingWalletImplementation: address(0x2) });
+
+        utils.writeL1AddressesFile(config, "./l1Addresses.json");
+
+        Utils.L1AddressesConfig memory configReadFromFile = utils.readL1AddressesFile("./l1Addresses.json");
+
+        assertEq(configReadFromFile.L1LiskToken, config.L1LiskToken);
+        assertEq(configReadFromFile.L1VestingWalletImplementation, config.L1VestingWalletImplementation);
+    }
+
+    function test_readAndWriteL2AddressesFile() public {
+        Utils.L2AddressesConfig memory config = Utils.L2AddressesConfig({
+            L2ClaimContract: address(0x01),
+            L2ClaimImplementation: address(0x02),
+            L2Governor: address(0x03),
+            L2GovernorImplementation: address(0x04),
+            L2LiskToken: address(0x05),
+            L2LockingPosition: address(0x06),
+            L2LockingPositionImplementation: address(0x07),
+            L2Reward: address(0x09),
+            L2RewardImplementation: address(0x10),
```

Comment:
Although this approach also works and makes no difference in testing, seems that 0xa, 0xb, ... 0xf are skipped between 0x9 and 0x10.
Either:
1. continue with `0xa` instead of `0x10`
2. Use decimal like `address(0)`, `address(1)`
3. 
```
uint160 index;
L2ClaimContract: address(index++),
L2ClaimImplementation: address(index++),
...
```